test/tests.js

```diff
@@ -0,0 +1,69 @@
+import isYes from '../src/isYes.js';
+
+const test = QUnit.test;
+
+test('yes is yes', function(assert){
+
+    const text = 'yes';
+    const expected = true;
+
+    const result = isYes(text);
+
+    assert.equal(result, expected);
+
```

Comment:
You don't need these blank lines at the beginning and end of a function. Since the rest of the spacing throughout your code is solid I'm not going to dock a point, but just be aware of this in the future!